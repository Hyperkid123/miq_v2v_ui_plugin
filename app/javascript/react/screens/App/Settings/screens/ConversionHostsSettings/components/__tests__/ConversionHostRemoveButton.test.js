// TODO

it('works', () => expect(true).toBe(true));
