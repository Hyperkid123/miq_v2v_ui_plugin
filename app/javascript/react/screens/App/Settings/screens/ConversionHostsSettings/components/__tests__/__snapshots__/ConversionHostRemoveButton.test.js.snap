// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`conversion host remove button renders correctly 1`] = `
<Button
  active={false}
  block={false}
  bsClass="btn"
  bsStyle="default"
  disabled={false}
  id="remove_12345"
  onClick={[Function]}
>
  Remove
</Button>
`;
